projects/common/vk_renderer.cpp

```diff
@@ -210,8 +210,12 @@ bool InitVulkan(VulkanRenderer* pRenderer, bool enableDebug, const VulkanFeature
 
         std::vector<const char*> enabledExtensions = {
             VK_KHR_SWAPCHAIN_EXTENSION_NAME,
-            VK_KHR_DYNAMIC_RENDERING_EXTENSION_NAME,
-            VK_EXT_DESCRIPTOR_BUFFER_EXTENSION_NAME};
```

Comment:
`VK_EXT_DESCRIPTOR_BUFFER_EXTENSION_NAME` and the associated can't be removed yet. It will break everything else that currently depends on it. Which is a lot of stuff.

Reply:
By default the pRenderer->Features.EnableDescriptorBuffer is true, so the extension gets added for every other Vulkan experiment. At least until you set the feature flag to false.

Reply:
My bad...should've read some moar lines down!